caikit_nlp/modules/text_generation/peft_prompt_tuning.py

```diff
@@ -1150,34 +1155,6 @@ def _execute_train_loop(
                     )
         return {"loss": training_loss_tracker}
 
-    @classmethod
-    def _filter_params_for_prompt_config(cls, prompt_config, params):
-        """Utility function to filter out required parameters for prompt_config
```

Comment:
I think this belongs in peft_config as it a utility to create any Config. I also think `create_hf_tuning_config(` in thsi file belongs there, but I did not move it because I dont know the rational behind heeping `create_hf_tuning_config` here in prompt_tuning.py

Reply:
It's more of a legacy reason than anything, all of the code for tuning config stuff was originally written in this file as part of this module, and the refactoring to pull some of it out happened later. I don't think there was a deep reason for leaving `create_hf_tuning_config` here